backend/Repositories/SightingRepo.cs

```diff
@@ -24,5 +27,25 @@ public IEnumerable<Sighting> GetAllSightings()
                 .Sightings
                 .Include(s => s.Species);
         }
+
+         public Sighting GetById(int id)
+        {
+            return _context
+                .Sightings
+                .Single(s => s.Id == id);
+        }
+         public Sighting DeleteSighting(int id)
```

Comment:
You don't need to call it `DeleteSighting` - it's the Sightings Repo, we know what we're deleting! `DeleteById` is probably a better name (and is consistent with your other method, `GetById`)